lib/chromiumRebaseL10n.js

```diff
@@ -2,11 +2,18 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+const path = require('path')
 const config = require('../lib/config')
 const util = require('../lib/util')
 const {rebaseBraveStringFilesOnChromiumL10nFiles, braveAutoGeneratedPaths} = require('./l10nUtil')
 
 const chromiumRebaseL10n = async (options) => {
+  // Revert to original before string replacement for android_chrome_strings.grd
+  // because original grd is overwritten with replaced one in ./src/brave.
```

Comment:
With this, we don't need to to change https://cs.chromium.org/chromium/src/chrome/android/BUILD.gn?l=155.